src/librustc_mir/const_eval.rs

```diff
@@ -474,8 +475,12 @@ impl<'mir, 'tcx> interpret::Machine<'mir, 'tcx> for CompileTimeInterpreter<'mir,
 
     /// Called immediately before a stack frame gets popped.
     #[inline(always)]
-    fn stack_pop(_ecx: &mut InterpCx<'mir, 'tcx, Self>, _extra: ()) -> InterpResult<'tcx> {
-        Ok(())
+    fn stack_pop(
+        _ecx: &mut InterpCx<'mir, 'tcx, Self>,
+        _extra: (),
+    ) -> InterpResult<'tcx, StackPopInfo> {
+        // Const-eval mode does not support unwinding from panics
+        Ok(StackPopInfo::Normal)
```

Comment:
This looks like we could make it a default impl in the `Machine` trait so that machines without unwinding support don't have to care at all?